packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/react/configmid.md

```diff
@@ -1,6 +1,6 @@
-The Okta React SDK requires an instance of an `OktaAuth` object with configuration properties. You need to set the `clientId` and `issuer` properties with the values you got from the CLI earlier. This can happen by directly setting the properties, with variable replacement that happens as part of the build process, or during application load time.
+The Okta React SDK requires an instance of an `OktaAuth` object with configuration properties. Set the `clientId` and `issuer` properties with the values that you got from the CLI earlier. This can happen by directly setting the properties, with variable replacement that happens as part of the build process, or during application load time.
```

Comment:
```suggestion
The Okta React SDK requires an instance of an `OktaAuth` object with configuration properties. Set the `clientId` and `issuer` properties with the values that you got from the CLI earlier. This can happen by directly setting the properties, with variable replacement that happens as part of the build process, or during app load time.
```

Reply:
Maybe "while an app is loading" instead of "during app load time" (assuming the meaning is the same)?

Reply:
Alisa is good with this update. Thanks!